kotlin-analysis/src/main/kotlin/org/jetbrains/dokka/analysis/CoreKotlinCacheService.kt

```diff
@@ -14,13 +15,6 @@ class CoreKotlinCacheService(private val resolutionFacade: DokkaResolutionFacade
         return resolutionFacade
     }
 
-    override fun getResolutionFacade(
-        elements: List<KtElement>,
-        platform: org.jetbrains.kotlin.platform.TargetPlatform
-    ): ResolutionFacade {
-        return resolutionFacade
-    }
```

Comment:
It seems like this method was just renamed. [Commit](https://github.com/JetBrains/intellij-community/commit/bacc42efe4ee07a4ec583ccfab042a40a36feb82#diff-7e143159c89cca57c82604e5d55a2da0cda5ebfe428b63bbfe8f0eeecc0423a6R99) here